packages/amplication-server/src/core/prismaSchemaParser/helpers.ts

```diff
@@ -102,11 +103,16 @@ export function idField(field: Field) {
   }
 }
 
-export function lookupField(field: Field) {
-  const fieldLookupType = field.attributes?.some(
-    (attribute) => attribute.name === "relation"
+export function lookupField(schema: Schema, field: Field) {
+  const models = schema.list.filter(
+    (item) => item.type === MODEL_TYPE_NAME
+  ) as Model[];
```

Comment:
It would be better to walk the schema once to get all models and pass those instead of `.list.filter` in multiple functions.

Reply:
will be handled here:
https://github.com/amplication/amplication/issues/6599